econml/bootstrap.py

```diff
@@ -120,19 +434,62 @@ def call(lower=5, upper=95):
                     return call_with_bounds(can_call, lower, upper)
                 return call
 
+        def get_inference():
+            # can't import from econml.inference at top level without creating mutual dependencies
+            from .inference import InferenceResults
+
+            prefix = name[: - len("_inference")]
+            if prefix in ['const_marginal_effect', 'effect']:
```

Comment:
Why not also marginal_effect? Is there another way of doing this that avoids keeping track of all the methods that each cate estimator is implementing. Maybe for now it's ok to just add marginal_effect to this list.

Reply:
Fixed by adding marginal effect explicitly; the issue is that the logic for how to display coefficients vs. effects vs. intercepts is different and we don't have any good way to know which one to use generically.